webview/src/experiments/components/AddStage.tsx

```diff
@@ -0,0 +1,24 @@
+import { MessageFromWebviewType } from 'dvc/src/webview/contract'
+import React from 'react'
+import styles from './table/styles.module.scss'
+import { IconButton } from '../../shared/components/button/IconButton'
+import { Add } from '../../shared/components/icons'
+import { sendMessage } from '../../shared/vscode'
+
+export const AddStage: React.FC = () => (
+  <div className={styles.addConfigButton}>
+    <p>Want to easily and efficiently reproduce your experiments? </p>
+    <IconButton
+      icon={Add}
+      onClick={() =>
+        sendMessage({ type: MessageFromWebviewType.ADD_CONFIGURATION })
+      }
+      text="Add a pipeline stage"
```

Comment:
[Q] Should this be title case? I think the rest of our buttons text use that.

Reply:
I was asking myself the same. I'll check and change.